cli/args/deno_json.rs

```diff
@@ -8,62 +8,6 @@ use deno_semver::jsr::JsrDepPackageReq;
 use deno_semver::jsr::JsrPackageReqReference;
 use deno_semver::npm::NpmPackageReqReference;
 
-#[cfg(test)] // happens to only be used by the tests at the moment
-pub struct DenoConfigFsAdapter<'a>(
```

Comment:
The benefit of `sys_traits` is that we can remove all these adapter and fs implementation structs.